room-battle.js

```diff
@@ -24,8 +24,9 @@ const MAX_TURN_TIME = 150;
 const STARTING_TIME_CHALLENGE = 280;
 const MAX_TURN_TIME_CHALLENGE = 300;
 
-const NOT_DISCONNECTED = 100;
 const DISCONNECTION_TICKS = 13;
+const DISCONNECTION_BANK_TICKS = 60;
```

Comment:
Wait, the disconnection bank is 5 minutes?

Reply:
300 seconds, as requested by the TDs

https://www.smogon.com/forums/threads/the-smogtours-timer.3637042/